### PR TITLE
Update README.md with proposal instructions and create directory stru…

### PR DESCRIPTION
**Pull request by jlongever**
@RackHD/corecommitters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+https://github.com/RackHD/specs/proposed
+
+Each functional specification should contain the following:
+
+* Subject: Simple proposal reference subject name (i.e. some-new-feature.md)
+* Background: A background description of the needed feature
+* Goals: Discrete (actionable, achievable) goals associated with the need
+* API: Interfaces, if relevant, or otherwise notes on coordination points between components and how they'll work together
+
+All design proposals will be reviewed by the core committers. Discussion between the core committers, submitter and all public contributors will take place via mailing list with topic <b>"RE: \<Subject\>"</b>
```
Comment (benbp) [inline README.md]: Do we even have a reference to what "core committers" means for anyone not internal? Perhaps a small descriptive blurb is warranted (e.g. "All design proposals will be reviewed by the maintainers of the RackHD project" ?). @heckj how should we refer to ourselves publicly?

Comment (benbp) [inline README.md]: Also, in markdown you can do **bolding** with `**bold**` instead of `<b>` tags.

File: proposed/example.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<b>New Design Proposal</b>
```
Comment (benbp) [inline proposed/example.md]: `**bolding**`

File: README.md (hunk 1 above)
Comment (jlongever) [inline README.md]: which is better?

Comment (jlongever) [inline README.md]: Core committers is referenced in other sections of the docs,  but maybe we should refer to it as the "code maintainers" ?

Comment (benbp) [inline README.md]: I think using the markdown syntax is preferable to HTML tags, though perhaps only for aesthetic reasons, so feel free to ignore me if you think I'm being pedantic :P

Comment (benbp) [inline README.md]: yeah, maintainers or some derivative seems like it would have a more obvious meaning to me if I was an external viewer.

Comment (jlongever) [inline README.md]: hah, i'll use **bold** to keep it consistent with md

**Comment by heckj** [comment]
refer to yourselves either way - "core committers" is a term used in other open source projects, especially where there's a large contribution base (I stole the term from the hadoop project actually), so I don't think it'll surprise anyone, but "RackHD maintainers" is fine too. :-) Whatever you want to call yourselves (within some reason - I know better than to make sweeping statements like that)

**Comment by benbp** [comment]
Oh, didn't realize you stole the term, thought you conjured it like with "functional specs" haha. Core Committers is fine for me if it's obvious to outsiders (I had just never heard of it before).

**Comment by jlongever** [comment]
too late,  already changed to "code maintainers"

**Comment by jlongever** [comment]
back to core committers... keep in consistent with the other docs.

**Comment by benbp** [comment]
:+1:

**Comment by jlongever** [comment]
:+1:

**Pull request merged**